src/test/java/com/researchspace/webapp/integrations/fieldmark/FieldmarkRealConnectionMVCIT.java

```diff
@@ -0,0 +1,67 @@
+package com.researchspace.webapp.integrations.fieldmark;
+
+import static com.researchspace.service.IntegrationsHandler.FIELDMARK_APP_NAME;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.researchspace.api.v1.controller.API_MVC_TestBase;
+import com.researchspace.api.v1.controller.API_VERSION;
+import com.researchspace.model.User;
+import com.researchspace.model.oauth.UserConnection;
+import com.researchspace.model.oauth.UserConnectionId;
+import com.researchspace.service.UserConnectionManager;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.web.servlet.MvcResult;
+
+// @Ignore("We leave the test Ignored so we can potentially run it manually "
+//    + "by pasting the bearer token")
```

Comment:
Could we run this test nightly i.e. `@RunIfSystemPropertyDefined(value = "nightly")`? I think we do that with other integrations "real" connection tests

Reply:
I think the way to go here is to mark as `@Ignore` on a merge to `rspace-os/rspace-web`, and potentially enable them in other repo that is run by our internal infrastructure which has the secrets.

Reply:
Yeah that makes sense. We could also pass the token as a maven param via a Jenkins secret then we don't need to have the token part of any codebase, like this https://stackoverflow.com/questions/57420331/pass-credentials-to-maven-from-jenkins